UI/src/components/widgets/deploy/view.html

```diff
@@ -1,5 +1,28 @@
 <div class="deploy-view">
     <div hide-empty="['{{ deployView.environments[0] }}']">
+        <style>
```

Comment:
This needs to go into it's on file. Create a `deploy.less` You can follow the example from the build widget

Reply:
On it!